.env.example

```diff
@@ -0,0 +1,3 @@
+POSTGRES_USER=<postgres_user>
+POSTGRES_PASSWORD=<postgres_password>
+DATABASE_URL=postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@postgres:5432/
```

Comment:
Please specify the database name.

Reply:
7692ee685afe3ee1c768dd6ea27d9f2729a00cc8